docs/components/identity/authorization.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: authorization
 title: Authorization
-description: "Learn how to manage authorizations in your Orchestration cluster."
+description: "Learn how to manage authorizations in your Orchestration Cluster."
```

Comment:
When we are referring to someone's "personal" orchestration cluster, i.e., "your Orchestration Cluster" -- do we want to lowercase?